Lamina/include/GL/camera.hpp

```diff
@@ -0,0 +1,145 @@
+#ifndef LM_CAMERA
+#define LM_CAMERA
+
+#include "../Lamina/include/Math/vector.hpp"
+#include "../Lamina/include/GL/window.hpp"
+#include "../Lamina/include/Math/constants.hpp"
+#include "../Lamina/include/user_interface/event.hpp"
+
+namespace lm
+{
+	static lm::mat4 LookAtMatrix(vec3f position, vec3f target, vec3f up)
```

Comment:
This function probably should be static. If you want to define it in the header, you can declare it as inline instead. Otherwise, you don't need any special function type and can declare it normally in the header and define it in the source file.